src/main.cpp

```diff
@@ -1233,6 +1155,111 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
     return res;
 }
 
+// TODO: move to policy/
+bool HasConflicts(CTxMemPool& pool, const CTransaction& tx, set<uint256>& setConflicts) {
```

Comment:
Another possibility is to have a GetConflicts method in the mempool and an AcceptConflictsAsReplaceable() function in policy. Or even better, a function AcceptTxReplacement with all the other relay/mining policy stuff that uses setConflicts (though the latter may be more complex).